api/bazel/repository_locations.bzl

```diff
@@ -88,4 +88,15 @@ REPOSITORY_LOCATIONS_SPEC = dict(
         release_date = "2020-08-17",
         use_category = ["api"],
     ),
+    com_github_apache_skywalking_data_collect_protocol = dict(
+        project_name = "SkyWalking API",
+        project_desc = "SkyWalking's language independent model and GRPC Api Definitions",
+        project_url = "https://github.com/apache/skywalking-data-collect-protocol",
+        version = "8.1.0",
+        sha256 = "ebea8a6968722524d1bcc4426fb6a29907ddc2902aac7de1559012d3eee90cf9",
+        strip_prefix = "skywalking-data-collect-protocol-{version}",
+        urls = ["https://github.com/apache/skywalking-data-collect-protocol/archive/v{version}.tar.gz"],
+        release_date = "2020-07-29",
+        use_category = ["api"],
```

Comment:
The repository being pulled in here only contains proto definitions. If this information is really for the Envoy API only it should likely be moved into `api/bazel/repository_locations.bzl`. For example the Zipkin and OpenCensus API information is in `api/bazel/repository_locations.bzl`

If that is not the case I'd suggest the `use_category` should be `observability_ext` (similar to Datadog, Lightstep, OpenTracing etc.). In addition the following section should be added: `extensions = ["envoy.tracers.skywalking"],`. My guess it should be moved to `/api/` - @htuch ?

Reply:
I am a little confused. In fact, this repository is already in the `api/bazel/repository_locations.bzl` just like `ZipKin`.

Reply:
You are correct. Ignore my comments